File: programmers/Lv_1/KimHyunseo/문자열내림차순으로배치하기.cpp
#include <string>
#include <vector>
#include<algorithm>
#include <functional>

using namespace std;

string solution(string s) {
	sort(s.begin(), s.end(),greater<int>());
	string answer = s;
	return answer;
}

int main()
{
	string s = "Zbcdefg";
	solution(s);
}

File: programmers/Lv_1/JeongJunKwon/문자열 내림차순으로 배치하기.java
import java.util.*;

class Solution {
  public String solution(String s) {
      String answer = "";
      String[] pieces = new String[s.length()];
      for(int i = 0; i < s.length(); i++) {
          String piece = s.substring(i, i+1);
          pieces[i] = piece;
      }
      
      Arrays.sort(pieces);
      
      for(int i = s.length() - 1; i >= 0; i--) {
          answer += pieces[i];
      }
      
      return answer;
  }
}